packages/pwa-kit-dev/src/configs/babel/babel-config.js

```diff
@@ -57,5 +57,22 @@ export default {
             presets: [require('@babel/preset-env'), require('@babel/preset-react')],
             plugins: [require('babel-plugin-dynamic-import-node-babel-7')]
         }
-    }
+    },
+    ignore: [
+        function (filepath) {
+            // Ignore node_modules, but include @salesforce/extension-* and pwa-kit-extension-sdk
+            if (/node_modules/.test(filepath)) {
+                // Only ignore non-extension @salesforce packages
+                if (
+                    /node_modules\/(?!@salesforce\/pwa-kit-extension-sdk|@salesforce\/extension-)/.test(
+                        filepath
+                    )
+                ) {
+                    return true
+                }
+                return false
+            }
+            return false
+        }
```

Comment:
I asked AI to invert the logic to get rid of the nested conditions. Here is the output i got:

```
function (filepath) {
    // Return false if it's an allowed extension package
    if (/node_modules\/@salesforce\/(pwa-kit-extension-sdk|extension-)/.test(filepath)) {
        return false;
    }
    
    // Return true if it's in node_modules (excluding allowed packages handled above)
    if (/node_modules/.test(filepath)) {
        return true;
    }
    
    // Return false for all other files
    return false;
}
```